### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +27,44 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix compatibility issue with the all-day event in the current UI. [#662](https://github.com/rokwire/events-manager/issues/662)
 
 ## [2.4.0] - 2021-06-22
+<<<<<<< HEAD
```
Comment (sandeep-ps) [inline CHANGELOG.md:30]: Please fix the CHANGELOG. Thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -246,7 +295,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - References to AWS keys and variables.
 
+<<<<<<< HEAD
```
Comment (sandeep-ps) [inline CHANGELOG.md:298]: There are merge conflict dividers in here too. Thanks.

Comment (bingzhang) [reply CHANGELOG.md]: should be fine now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,15 +6,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ### Fixed
 - Fix user event image operation using event id of events building block.[#707](https://github.com/rokwire/events-manager/issues/707)
 
 ## [2.4.1] - 2021-06-23
+
 ### Fixed
 - Fix compatibility issue with the all-day event in the current UI. [#662](https://github.com/rokwire/events-manager/issues/662)
 
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:53]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,15 +6,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Split datetime-local control to date and time to fix compatibility issue with firefox and safari. [#479](https://github.com/rokwire/events-manager/issues/479)
+- Move Free Event above Cost Description. [#661](https://github.com/rokwire/events-manager/issues/661)
+
 ## [2.4.2] - 2021-08-11
 
 ### Fixed
 - Fix user event image operation using event id of events building block.[#707](https://github.com/rokwire/events-manager/issues/707)
 
 ## [2.4.1] - 2021-06-23
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:49]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,15 +6,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0] - 2021-09-07
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:10]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,15 +6,51 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Fix date filters in user event. [#728](https://github.com/rokwire/events-manager/issues/728)
+- Fix the wrong format of user input datetime.[#740](https://github.com/rokwire/events-manager/issues/740)
+
+### Changed
+- Hide Past event -> Hide Past Events. [#641](https://github.com/rokwire/events-manager/issues/641)
+- Crop event title at a fixed number of characters. [#667](https://github.com/rokwire/events-manager/issues/667)
+- Move dates under a second row below the title and move badges to the right of the title. [#666](https://github.com/rokwire/events-manager/issues/666)
+- Split datetime-local control to date and time to fix compatibility issue with firefox and safari. [#479](https://github.com/rokwire/events-manager/issues/479)
+- Move Free Event above Cost Description. [#661](https://github.com/rokwire/events-manager/issues/661)
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:41]:
```suggestion

### Removed
- User events that are not associated with groups from being viewable. [#678](https://github.com/rokwire/events-manager/issues/678)
```

Let's add one more entry as this is what will cause the breaking change.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**